rfc-095-long-term-future-apps.md

```diff
@@ -0,0 +1,90 @@
+# RFC 95: Long term future of applications
+
+## Summary
+
+This RFC describes the direction of travel for the applications that make up GOV.UK.
+
+## Problem
+
+We currently have 58 separate applications running on GOV.UK. The [Platform architecture principles][princ] explain where we're going from an architectural standpoint. This RFC is an attempt to explain the long term vision of each of GOV.UK's apps.
+
+The purpose of doing this is to make clear:
+
+- where to put new functionality
+- which applications need particular attention
+
+## Proposal
+
+| Name | Description | Rough action plan |
+| -- | -- | -- |
+| calculators | Serves the Child benefit tax calculator on GOV.UK | Merge into smart-answers |
+| calendars | Serves /bank-holidays and /when-do-the-clocks-change on GOV.UK | Merge into custom frontend |
+| ckanext-datagovuk | Extension for use with datagovuk_publish | Retire in favour of datagovuk_find/datagovuk_publish? |
+| collections-publisher | Publishes step by steps, /browse pages, and legacy /topic pages on GOV.UK | While topics and browse pages will be removed, this can still be the app for curating collections. We might move Whitehall's document collections into this app |
+| content-audit-tool | Deprecated application to audit content | Retire, as it's not used |
+| content-publisher | WIP - Future publisher of content on GOV.UK | Expand to publish most content |
+| datagovuk_find | Beta version of Find Data | Merge into finder-frontend? |
+| datagovuk_publish | Beta version of publish data | Retire in favour of content publisher? |
+| email-alert-service | Message queue consumer that triggers email alerts for GOV.UK | Merge into email-alert-api, as it's only task is to take things from the message queue and forward it there |
+| licence-finder | Serves licence pages on GOV.UK | Retire |
```

Comment:
It's unclear what the future is for licence-finder. We have some responsibility under the EU Service Directive to provide a central way for users to discover where they _can_ apply for licences. I'd say it's likely that licence-finder as-is will be retired, but additional functionality might be required in (e.g.) local links manager to support what we do need to do.